src/app/kbv/controllers/load-question.js

```diff
@@ -28,7 +28,7 @@ class LoadQuestionController extends BaseController {
   }
   next(req) {
     if (req.session?.question) {
-      return "question";
```

Comment:
Unfortunately the existing example in Experian KBVs isn't very clear or helpful.

https://github.com/govuk-one-login/ipv-cri-kbv-hmrc-front/pull/23/files shows a better approach:

*steps:*
```
{ fn: loadQuestionController.prototype.hasQuestion, next: "question" },
```

*controller:*
```
hasQuestion(req) {
    return req.session?.question !== undefined;
}
```

This makes is clearer in the steps file how the routing is being managed.